lib/questions.rb

```diff
@@ -187,12 +258,23 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  str_method()
```

Comment:
`send(str_method.to_sym)`
...but just checked yours and it worked, because it'll raise a NameError. The test needs to be better.